pkgs/top-level/all-packages.nix

```diff
@@ -4701,8 +4701,19 @@ with pkgs;
   cloudbrute = callPackage ../tools/security/cloudbrute { };
 
   cloudflared = callPackage ../applications/networking/cloudflared {
-    # https://github.com/cloudflare/cloudflared/issues/1054
-    buildGoModule = buildGo120Module;
+    # https://github.com/cloudflare/cloudflared/issues/1151#issuecomment-1888819250
+    buildGoModule = buildGoModule.override {
+      go = go.overrideAttrs {
+        pname = "cloudflare-go";
+        version = "unstable-2023-12-06";
```

Comment:
version must start with a number

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
Thanks! Resolved in `f13a5874518b`